### PR TITLE
[KL-515] Clear keychain on app reinstall

### PR DESCRIPTION
**Pull request by MuKoJla**

### CODE REVIEW

File: libs/ui/src/store/root-state.epics.ts [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 import { resetStore$ } from '../utils/keychain.utils';
 
-import { resetApplicationAction, untypedNavigateAction } from './root-state.actions';
+import { resetApplicationAction, resetKeychainOnInstallAction, untypedNavigateAction } from './root-state.actions';
+import { RootState } from './store';
+
+const resetKeychainOnInstallEpic = (action$: Observable<Action>, state$: Observable<RootState>) =>
+  action$.pipe(
+    ofType(resetKeychainOnInstallAction.submit),
+    withLatestFrom(state$, (_, state) => state.settings.isFirstAppLaunch),
```
Comment (thesvyatoslavtt) [inline libs/ui/src/store/root-state.epics.ts:17]: why do we reset the store on the install and not the reset wallet?

Comment (MuKoJla) [reply libs/ui/src/store/root-state.epics.ts:17]: It will work for both install and reset

File: libs/ui/src/store/root-state.epics.ts [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+    filter(isFirstAppLaunch => isFirstAppLaunch),
```
Comment (thesvyatoslavtt) [inline libs/ui/src/store/root-state.epics.ts:18]: do we need this filter?

Comment (MuKoJla) [reply libs/ui/src/store/root-state.epics.ts:18]: yes, we need to reset it, only when `isFirstAppLaunch` is true
image

File: apps/extension/package.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
     "react-dom": "^18.2.0",
     "react-native-web": "^0.18.10",
     "react-scripts": "^5.0.1",
+    "redux-state-sync": "^3.1.4",
```
Comment (unixvb) [inline apps/extension/package.json:28]: I think we don't need this dependency for `extension` if it used in `libs/ui`

Comment (MuKoJla) [reply apps/extension/package.json:28]: there is an error, if I delete it
image

**Review by unixvb — APPROVED** [review]

**Review by Boltman92 — APPROVED** [review]

**Review by thesvyatoslavtt — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]

**Pull request merged**